src/e2e-test/java/io/cdap/plugin/pubsub/stepsdesign/PubSubBase.java

```diff
@@ -51,4 +54,20 @@ public void enterPubSubTopicName(String topicName) {
   public void closeThePubSubProperties() {
     PubSubActions.close();
   }
+
+  @Then("Click on get schema")
```

Comment:
Remove this.

Use step from e2e framework `PipelineSteps `-> Click on the Get Schema button